packages/calypso-build/tsconfig.json

```diff
@@ -14,25 +14,7 @@
 		// Import non-ES modules as default imports.
 		"esModuleInterop": true,
 		"skipLibCheck": false,
-		"module": "esnext",
-		"baseUrl": ".."
+		"module": "esnext"
 	},
-	"exclude": [
-		"**/build-module/*",
-		"**/build-style/*",
-		"**/build/*",
-		"**/dist/*",
-		"**/node_modules/*",
```

Comment:
I wonder if these are general and should be kept?

Reply:
General enough for `calypso-build` probably! However:

> `files`, `include` and `exclude` from the inheriting config file _overwrite_ those from the base config file.

(https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#configuration-inheritance-with-extends)

So we'll still need to have them in the derived tsconfigs as well.

I'll add them to `calypso-build`'s tsconfig in a follow-up (I'd like to see if I can drop the wildcards).